app/views/pages/incompany.html.erb

```diff
@@ -1,2 +1,4 @@
 <h1>Incompany</h1>
-<%= render './shared/get_ready_for_the_day_inCompany' %>
+
+<%= render './shared/choose_your_course' %>
+<%= render './shared/get_ready_for_the_day_(inCompany)' %>
```

Comment:
Config your editor so it adds a single newline at the end of the file (see that red thing at the end of this line)